src/components/App.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
```

Comment:
the app component should be out in the /src folder since it will be used more than just a component

Reply:
it will no longer used to next tasks , it was used to prove that setup react project work fine